src/main/java/dao/GuestBookDao.java

```diff
@@ -0,0 +1,83 @@
+package dao;
+
+import java.util.List;
+
+import util.DaoHelper;
+import vo.Category;
+import vo.GuestBook;
+import vo.Party;
+import vo.User;
+
+public class GuestBookDao {
+	
+	private static GuestBookDao instance = new GuestBookDao();
+	private GuestBookDao() {}
+	public static GuestBookDao getInstance() {
+		return instance;
+	}
+	
+	// 방명록 등록하기
+	public void insertGuestBook (GuestBook guestbook) {
+		DaoHelper.update("guestBookDao.insertGbook", guestbook.getUser().getId(),
+													 guestbook.getParty().getNo(),
+													 guestbook.getContent());	
+	}	
+		
+	// 파티번호에 해당하는 방명록 조회하기
+	public List<GuestBook> getGuestBooksByNo(int partyNo){
```

Comment:
```suggestion
	public List<GuestBook> getGuestBooksByPartyNo(int partyNo){
```
방명록번호가 따로 있어서 Party를 꼭 붙여야 합니다.